Cluster Programs

```diff
@@ -1,3 +1,73 @@
+By looking at the SAS code.
+
+3) In the original dataset from Harvard Dataverse you can find the variables Invnum and Invnum_N. 
+Invnum: Initial inventor number: Patent + Invseq. 
+Invnum_N: Disambiguated inventor number. This function assign a number to each of the individual inventors, this way an inventor with more than one patent will have the same number.  
+In the program, we are using Invnum_N, which later it’s rename to Invnum.
+
+4) Next is presenting the differences between both method of matching, Thompson and Jaffe e.t, in order to determined the one we are using.  
+In Thompson 2005 article the different methods are made clear. 
+Thompson (2005) argue that Jaffe constructed three patents samples: a set of originating patents, a set of citing patents which reference one of the originated patents and a set of control patents matched to each citing patent.  
+Each control patent shared the same technology class it matched citing patent. The experiment compared the probabilities that the citing patent and its matched control patent, 
+were filed by inventors living in the same geographic location as the originating patent. 
+Thompson declares that, controlling for unobservables using matching methods is invariably a dangerous exercise because one can rarely be confident that the control are doing their job.  
+Imperfect matching may simply introduce noise and a corresponding loss of efficiency, but these are not the applications where matching is critical.  
+
+This next part is to check and defined which method are we using.  
+Thompson critics on JTHs method is that: 
+ 
+1) JTH control patents were selected using broad, three-digit technological classification codes of the U.S. Patent and Trademark Office (USPTO), a level of aggregation that surpasses considerable within-class heterogeneity.  
+This means that, if I match the citing patent with the originating patent using the three digits class, it is also likely to fall in the same subclass.  
+In contrast, the control patent is likely to be drawn from a different subclass, and it will fail to control for pre-existing geographic patterns of production.  
+2) Patents typically contain many distinct claims, to each of which a technological classification is assigned.  
+The particular claim in a citing patent that can be associated with a citation to prior art may be quite distinct from the claim that generated the corresponding control patent.  
+This doesn’t ensure the existence of any industrial link between the originating and control patent. 
+ 
+Thompson claims that these two features of the control selection process means there is no guarantee that the control patent has any industrial similarity either to the citing or to the originating patent.  
+
+Thompson’s criteria. 
+1)He select control patents using the technology subclass, a much finer level of disaggregation than the three-digit classification scheme.  
+Instead of using for example class 231, he uses 231.3 
```

Comment:
Can you say more about 4-digit versus 6-digit controls? In the robustness check we have in the working paper, we use 6-digit controls. I thought that subclasses were always three-digit numbers, just like nclasses, but I could easily be wrong